cmd/dmsg-server/commands/root.go

```diff
@@ -22,12 +26,18 @@ import (
 	"github.com/skycoin/dmsg/servermetrics"
 )
 
+const (
+	defaultDiscoveryURL = "http://dmsg-discovery:9090"
```

Comment:
Should probably point it to the production discovery.

Reply:
yes, will do